matlab/schemas/+vis/FancyBar.m

```diff
@@ -1,18 +1,20 @@
  %{
-vis.FancyBar (manual) # pink noise with periods of motion and orientation$
+vis.FancyBar (manual) # Moving Bar stimulus that keeps the size and speed
+of the bar constant relative to the mouse�s perspective. 
+The bar can be either a Flashing checkeboard (grating=0) or a Moving grating (grating=1)
 -> vis.Condition
 ---
 pre_blank                   : double                        # (s) blank period preceding trials
 luminance                   : float                         # (cd/m^2)
 contrast                    : float                         # michelson contrast
 bar_width                   : float                         # Degrees
 grid_width                  : float                         # Degrees
-bar_speed                   : float                         # Bar speed in �/s
+bar_speed                   : float                         # Bar speed in deg/s
 flash_speed                 : float                         # cycles/sec temporal frequency of the grid flickering
 grating                     : float                         # selection beween grating (1) or flashing (0)
 grat_width                  : float                         # in cycles/deg
 grat_speed                  : float                         # in cycles/deg
-axis                        : float                         # the axis of the bar movement
+axis                        : float                         # the direction of bar movement
```

Comment:
what are the units for axis?  degrees relative to up, clockwise?  up=0, right=90, down=180, left=270?